src/routes/index.js

```diff
@@ -12,67 +12,67 @@ import Users from 'pages/Users';
 const routes = [
   {
     path: routesPaths.index,
-    component: <Home />,
+    element: <Home />,
     exact: true,
-    private: true,
+    isPrivate: true,
   },
   {
     path: routesPaths.ranking,
-    component: <Ranking />,
+    element: <Ranking />,
     exact: true,
-    private: true,
+    isPrivate: true,
     title: 'Ranking',
   },
   {
     path: routesPaths.rules,
-    component: <Rules />,
+    element: <Rules />,
     exact: true,
-    private: true,
+    isPrivate: true,
     title: 'RULES',
     subtitle: 'Can you guess the word before other Rootstrappers do?',
   },
   {
     path: routesPaths.settings,
-    component: <Settings />,
+    element: <Settings />,
     exact: true,
```

Comment:
I think all the `exact`s are not being used, aren't they?

Reply:
you are right, thanks!